Xamarin.PropertyEditing/ISolidBrushPropertyInfo.cs

```diff
@@ -0,0 +1,10 @@
+using System.Collections.Generic;
+
+namespace Xamarin.PropertyEditing
+{
+	// TODO: Add support for predefined color values
+	public interface ISolidBrushPropertyInfo : IPropertyInfo
```

Comment:
Generally stuff like this has been done as "light-up" interfaces for the individual feature (color spaces, in this case), rather than as a whole property type. This prevents us from having to make a bunch of these if other brush types end up needing color spaces and prevents platforms that don't need to from having to implement it anyway.

Reply:
Good point, thanks.